### PR TITLE
Bugfix braces

### PR DESCRIPTION
**Pull request by mnabialek**
Target of https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3001 into 2.2 branch

### CODE REVIEW

File: src/Fixer/Basic/BracesFixer.php [hunk 1]
```diff
@@ -849,7 +849,7 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
             $previousToken = $tokens[$nextTokenIndex - 1];
             // do not indent inline comments used to comment out unused code
             if (
-                0 === strpos($nextToken->getContent(), '//'.$this->whitespacesConfig->getIndent())
+            (0 === strpos($nextToken->getContent(), '//'.$this->whitespacesConfig->getIndent()) || $nextToken->getContent() === '//')
```
Comment (SpacePossum) [inline src/Fixer/Basic/BracesFixer.php]: Is this indentation change a typo or the result of changes made in the fixer? I think this should be reverted.

Comment (mnabialek) [reply src/Fixer/Basic/BracesFixer.php]: It was typo, got fixed

**Comment by GrahamCampbell** [comment]
👍

**Comment by keradus** [comment]
Thank you @mnabialek.

**Pull request closed without merging**